### ISSUE TITLE
Trouble following packaging libraries tutorial

### ISSUE THREAD

**Issue by alessandroBombelli** [pypa/packaging-problems#297]
Thank you for providing feedback on Python packaging!

To help us help you, please fill out as much of the following as you can. If a question is not relevant, feel free to skip it.

1. What is your operating system and version?

macOS Mojave v10.14.6

2. What is your Python version?

python 3.7 via virtual environment

3. What version of pip do you have?

pip 19.2.3

4. If following an online tutorial or guide, please provide a link to the page or section giving you trouble:

https://packaging.python.org/tutorials/packaging-projects/#packaging-python-projects

5. Could you describe your issue in as much detail as possible?

Hello all. I am following the exact steps as stated in the webpage (link in point 4.), but when I run the code 
python3 setup.py sdist bdist_wheel 
the output is 
-bash: setup.py: command not found
I listed the files inside my folder, and setup.py is there. I am not sure what I am doing wrong. I am attaching a screenshot of my terminal with all the steps I take until I get the error

<img width="957" alt="Screen Shot 2019-11-17 at 11 50 08 AM" src="https://user-images.githubusercontent.com/55788224/69006606-87866200-0931-11ea-977d-9ec08208173d.png">

Basically, I am activating my virtual environment, checking which python I am using, navigating to the folder that contains the setup.py file, checking the file exists, and try to run it to no avail. Any help would be greatly appreciated! Thanks.

**Comment by merwok** [comment]
You ran `setup.py` instead of `python setup.py`

**Comment by alessandroBombelli** [comment]
> You ran `setup.py` instead of `python setup.py`

Thanks, that indeed worked!
